README.md

```diff
@@ -5,17 +5,19 @@ bringing you reproducible builds and better caching.
 
 This repository hosts two components:
 
-- a [Nixpkgs](https://github.com/NixOS/nixpkgs) overlay, located at `overlay`
+- A [nixpkgs](https://github.com/NixOS/nixpkgs) overlay, located at the `/overlay`
   directory, providing utilities to build your Cargo workspace and set up
   airplane mode.
   
-- a utility written in Rust to generate version pins of dependency crates
+- A utility written in Rust to generate version pins of crate dependencies
   and boilerplates
+  
+Together, these components will take an existing Cargo.lock and delegate the process of fetching and combining your dependencies (generated by cargo) using the deterministic nix package manager.
 
 
 ## Build instructions
 
-1. Install the [Nix package manager](https://nixos.org/nix) on your *NIX
+1. Install the [nix package manager](https://nixos.org/nix) on your *nix
```

Comment:
The package manager name "nix" should be capitalized.

Reply:
Done :)